src/components/GlobalChart/index.js

```diff
@@ -56,38 +62,26 @@ const GlobalChart = ({ display }) => {
   }, [dailyData, utcStartTime, volumeWindow, weeklyData]);
   const below800 = useMedia('(max-width: 800px)');
 
-  // update the width on a window resize
-  const ref = useRef();
```

Comment:
Why did you remove this? It doesn't take effect on the width?

Reply:
Yes, it had no effect on resize.